### PR TITLE
fra + frp

### PR DESCRIPTION
**Pull request by hectoralos**
Added the missing strings in fra.json and the new frp language (Arpitan/Francoprovençal).

### CODE REVIEW

File: assets/strings/README.md [hunk 1]
```diff
@@ -3,15 +3,17 @@ Localization
 
 This directory contains the JSON files powering Html-tools' localization as well as some helpful utilities. `locales.json` contains a reference to each JSON strings file as well as each language's endonym. Localization instructions are available on the [Apertium Wiki](http://wiki.apertium.org/wiki/Apertium-html-tools).
 
-After editing localizations, please run `make` and commit its changes.
```
Comment (sushain97) [inline assets/strings/README.md:6]: Could you undo this change?

**Review by sushain97 — CHANGES_REQUESTED** [review]
This is great! Just one thing that I think you accidentally changed.

**Comment by hectoralos** [comment]
Hopefully I've done it correctly. It seems that I didn't pull the last version before introducing the changes in the file.

**Comment by sushain97** [comment]
I think your new changes overwrote the wrong file: https://github.com/apertium/apertium-html-tools/pull/382/files

**Comment by hectoralos** [comment]
I close this request I begin another from the scratch.

**Pull request closed without merging**